vuepress/docs/next/tutorials/create/pb/export-bundle-from-application.md

```diff
@@ -71,14 +66,34 @@ mkdir bundle
 ```
 2. Run the bundler:
 ``` sh
-ent bundler from-env --location bundle --code YOUR-TEST-BUNDLE --description “Your Test Bundle”
+ent bundler from-env --location bundle --code YOUR-BUNDLE-NAME --description “Your Exported Bundle”
 ```
-The bundler will inspect the application using Entando APIs, collect information about individual components, construct appropriate descriptor files and assemble the top-level descriptor file.
+The bundler will inspect the application using Entando APIs, collect information about individual components, construct appropriate descriptor files, and assemble the top-level descriptor file.
 
 ``` 
 $ ls bundle
 assets      contentModels  contents         fragments  labels     pageModels  resources
 categories  contentTypes   descriptor.yaml  groups     languages  pages       widgets
 ```
 
-You now have a complete Entando project structure! You can inspect the output to edit the exported components or deploy it to another Entando Application.
+You now have a complete Entando project structure for a git-based bundle! You can inspect the output to edit the exported components or [deploy](publish-project-bundle.md) it to another Entando Application. 
+::: tip Note:
+To convert this project to a docker-based bundle, continue with the steps below. 
+:::
+
+### Construct the Docker-based Bundle Structure
+
+1. Initialize a new bundle and switch to that directory:
+``` 
+ent bundle init YOUR-BUNDLE-NAME
+cd YOUR-BUNDLE-NAME
+```
+2. Copy the resources from the bundle directory, with the exception of microservices (/plugins) and micro frontends (/widgets), to the `platform` directory:
+```
+cp -R ../testBundle/bundle/assets platform/
+```
+Repeat for all the other components in your bundle, including assets, contentModels, contents, fragments, labels, pageModels, resources, categories, contentTypes, groups, languages, and pages.
+
+If there are micro frontends or microservices in the bundle, use the source code to migrate them to their designated folders. Use the [ent bundle CLI tool](../../../docs/getting-started/ent-bundle.md) to assist in the process. Also check out the [Entando Bundle details](../../../docs/curate/bundle-details.md) page to define specific attributes for the MFEs and MSs in the bundle descriptor, `entando.json`.
```

Comment:
the use of the source code here needs some clarification... do you make use of the source code when migrating, e.g. move the source code, or do you use the source code as in execute it? i think it's the former...

@nshaw i feel like there was a decision regarding the use of the word "migration" - that it should be used exclusively wrt changing platforms?

did we introduce MS as an acronym here? MSs looks really odd and is not a universally adopted initialism. even a somewhat exhaustive google search doesn't yield microservice as a candidate

Reply:
vague is fine (no details necessary), but it's not clear how to read it and that's a bit different

sure. i remember at one point we were explicitly defining uncommon acronyms before we used them. in this case it should be pretty obv what MS stands for given the context

Reply:
"1 & 2 edited a bit but it's intentionally vague" 

there may be many ways to do the same thing but that's beside the point here. this clearly reads like you run source code to perform a migration.